diff --git a/node_modules/warp-contracts/lib/cjs/core/modules/impl/ArweaveGatewayInteractionsLoader.js b/node_modules/warp-contracts/lib/cjs/core/modules/impl/ArweaveGatewayInteractionsLoader.js
index bbcc21b..1f70e49 100644
--- a/node_modules/warp-contracts/lib/cjs/core/modules/impl/ArweaveGatewayInteractionsLoader.js
+++ b/node_modules/warp-contracts/lib/cjs/core/modules/impl/ArweaveGatewayInteractionsLoader.js
@@ -10,7 +10,7 @@ const TagsParser_1 = require("./TagsParser");
 const MAX_REQUEST = 100;
 function bundledTxsFilter(tx) {
     var _a, _b;
-    return !((_a = tx.node.parent) === null || _a === void 0 ? void 0 : _a.id) && !((_b = tx.node.bundledIn) === null || _b === void 0 ? void 0 : _b.id);
+    return tx.node.block !== null && !((_a = tx.node.parent) === null || _a === void 0 ? void 0 : _a.id) && !((_b = tx.node.bundledIn) === null || _b === void 0 ? void 0 : _b.id);
 }
 exports.bundledTxsFilter = bundledTxsFilter;
 class ArweaveGatewayInteractionsLoader {
